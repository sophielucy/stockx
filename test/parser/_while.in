while ( x < 42 ) {
    print("42 is my favorite number");
}
