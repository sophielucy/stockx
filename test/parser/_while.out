While(Binop(Id(x), Less, IntLiteral(42))) { Block(Call(print, StringLiteral(42 is my favorite number))) }
